activestorage/app/models/active_storage/blob/representable.rb

```diff
@@ -17,6 +17,10 @@ module ActiveStorage::Blob::Representable
   #
   #   avatar.variant(resize_to_limit: [100, 100]).processed.url
   #
+  #   or if you are using pre-defined variants
```

Comment:
Code examples have 2 extra spaces of indentation. As this is regular documentation, these spaces extra spaces should be removed:
```suggestion
  # or if you are using pre-defined variants
```